app/sidekiq/lighthouse/form526_document_upload_polling_job.rb

```diff
@@ -41,32 +44,49 @@ class Form526DocumentUploadPollingJob
     end
 
     def perform
-      Lighthouse526DocumentUpload.pending.status_update_required.in_batches(
+      successful_documents_before_polling = Lighthouse526DocumentUpload.completed.count
```

Comment:
Do we have any fear that this may grow to take forever if there is tons of these (which there will be)? It should just be doing a count, so probably would be fine?

Reply:
@NB28VT any comment here?

Reply:
Sorry @rmtolmach we should have discussed us here instead of in Slack. @kylesoskin as discussed you think this is good to go yeah?

Thanks for calling this out though @rmtolmach. TLDR we don't think this will be a problem, at least for a little while and they are hoping we can ship all of this soon thanks!

Reply:
thanks!